### PR TITLE
Create Rule “test-rule-jake/rule”

### PR DESCRIPTION
**Pull request by jakebayliss**
Hi Tiago, this is just my sugarlearning task to create a new rule :)

### CODE REVIEW

File: rules/test-rule-jake/rule.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+type: rule
+title: Induction Task - Do you know this is a test rule
```
Comment (tiagov8) [inline rules/test-rule-jake/rule.md:3]: missing question mark

File: rules/test-rule-jake/rule.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  * Images
+* Make yourself the author
+* Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec scelerisque, metus eu iaculis suscipit, sem mauris gravida nunc, ac dictum nisl felis non nulla. Curabitur in orci nisi.
+
+### Link
+
+[I'm an inline-style link](https://www.google.com)
+
+### Image
+
```
Comment (tiagov8) [inline rules/test-rule-jake/rule.md:38]: Can you add the ::: good markdown?

File: rules/test-rule-jake/rule.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+![Figure: Good figure](https://images.unsplash.com/photo-1491472253230-a044054ca35f?ixlib=rb-1.2.1&auto=format&fit=crop&w=1000&q=80)
+
+### Video
+
+This is a test video\
```
Comment (tiagov8) [inline rules/test-rule-jake/rule.md:43]: remove backslash

File: rules/test-rule-jake/rule.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+`youtube: https://www.youtube.com/embed/0ugMkda9IBw`
+
+### Email Template
+
+::: email-template\
```
Comment (tiagov8) [inline rules/test-rule-jake/rule.md:48]: remove backslash

File: rules/test-rule-jake/rule.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum tincidunt ac eros lobortis cursus. Vivamus vehicula sodales dolor ut ultricies. Sed faucibus tortor ut mauris venenatis, vitae vestibulum eros laoreet. Aenean convallis molestie justo in dignissim. Sed elementum lacus condimentum erat vehicula dapibus. Etiam vulputate vestibulum semper. 
+
+Vivamus sed aliquet urna.
+
+:::\
+:::
+
+### Greybox
+
+:::greybox
```
Comment (tiagov8) [inline rules/test-rule-jake/rule.md:68]: missing space

File: rules/test-rule-jake/rule.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+  - title: Jake Bayliss
+    url: https://www.ssw.com.au/people/jake-bayliss
+created: 2022-05-12T05:07:48.559Z
+guid: 73218c8b-b8d6-4d83-91f9-ae06cc9786ba
+---
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec scelerisque, metus eu iaculis suscipit, sem mauris gravida nunc, ac dictum nisl felis non nulla. Curabitur in orci nisi.
+
+<!--endintro-->
+
+# Rules for creating test rules
```
Comment (tiagov8) [inline rules/test-rule-jake/rule.md:15]: I would avoid heading 1 inside the content... not a mistake, but h1 is normally used for the title

File: rules/test-rule-jake/rule.md (hunk 1 above)
Comment (jakebayliss) [reply rules/test-rule-jake/rule.md]: Added

File: rules/test-rule-jake/rule.md (hunk 2 above)
Comment (jakebayliss) [reply rules/test-rule-jake/rule.md]: Added

File: rules/test-rule-jake/rule.md (hunk 6 above)
Comment (jakebayliss) [reply rules/test-rule-jake/rule.md]: Changed to heading 2

**Comment by tiagov8** [comment]
Test pass!

**Pull request closed without merging**